draft-ietf-quic-multipath.md

```diff
@@ -1422,7 +1422,8 @@ in {{QUIC-TRANSPORT}}.
 
 This document does not specify per-path idle timeouts. An endpoint
 can decide to close a path at any time, whether the path is in active
-use or not, by sending a PATH_ABANDON frame. It is not required
+use or not, by sending a PATH_ABANDON frame. For the initial PATH_ABANDON
+frame to close a path, it is not required
 to send a PATH_ABANDON frame at any specific point in time.
 For example, an endpoint may wait until it will anyway send another frame.
```

Comment:
Based on https://github.com/quicwg/multipath/issues/505#issuecomment-2727266283, shouldn't we explicitly state that we want endpoints to immediately react to the reception of a PATH_ABANDON? Also the normative nature of the should depends if we keep this in implementation guidance or not...

```suggestion
However, the endpoint that did not initiated the path abandonment should immediately
reply with a PATH_ABANDON frame upon reception of a PATH_ABANDON frame (see {{path-close}}).

```

Reply:
We can add this sentence if that makes it even more clearer. However why did you remove the other sentence?

Reply:
Oh, that was an oversight, will fix this!